### PR TITLE
docs(material/chips): add chips avatar sample

### PR DESCRIPTION
**Pull request by AlirezaEbrahimkhani**
resolves angular#22905

### CODE REVIEW

File: src/components-examples/material/chips/chips-avatar/chips-avatar-example.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<mat-chip-list aria-label="Dog selection">
+  <mat-chip>
+    <mat-chip-avatar
```
Comment (andrewseguin) [inline src/components-examples/material/chips/chips-avatar/chips-avatar-example.html:3]: Change the ending `>` tag markers to be on the same line as their opening

**Comment by AlirezaEbrahimkhani** [comment]
@andrewseguin 
Hi ! I made the changes you mentioned

File: src/components-examples/material/chips/chips-avatar/chips-avatar-example.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<mat-chip-list aria-label="Dog selection">
+  <mat-chip>
+    <mat-chip-avatar>
+        <img src="https://material.angular.io/assets/img/examples/shiba1.jpg" alt="shiba1 image"/>
```
Comment (andrewseguin) [inline src/components-examples/material/chips/chips-avatar/chips-avatar-example.html:4]: lets change alt to `alt="Photo of a Shiba Inu"`

**Comment by andrewseguin** [comment]
Looks good - one more thing, you'll be able to make the linter happy if you use `material/chips` the scope of your change:

`docs(material/chips): add chips example with avatars`

**Comment by AlirezaEbrahimkhani** [comment]
@andrewseguin 
Thanks for the full review.
I made the changes you mentioned.

**Pull request merged**